canonical-web-and-design/security.ubuntu.com
# Init project

Initialising the API with an `hello-world` endpoint and generate the `spec` and `docs` of the project.

## QA

QA for code:
- clone repo
- fetch changes
- docker-compose up -d
- dotrun
- go to http://0.0.0.0:8030/docs and try the endpoint; it should work

## Issue

Fixes
